source/bdDetect.py

```diff
@@ -655,3 +666,8 @@ def addDeviceScanner(
 		scanForDevices.register(scanFunc)
 		if moveToStart:
 			scanForDevices.moveToEnd(scanFunc, last=False)
+
+
+class DeviceMathcer:
+    fallBackDevices = list()
+    matches = list()

```

Comment:
Can you add the type to the lists here to replace `?`

```suggestion
    fallBackDevices: List[?] = []
    matches: List[?] = []
```